elements/rh-secondary-nav/rh-secondary-nav.ts

```diff
@@ -436,6 +416,14 @@ export class RhSecondaryNav extends LitElement {
       this.style.setProperty('--_chevron-transform-expanded', 'rotate(var(--_chevron-down-angle)) translate(var(--_chevron-thickness), calc(-1 * var(--_chevron-thickness)))');
```

Comment:
Think we could set this on a shadow container?

Reply:
@bennypowers https://github.com/RedHat-UX/red-hat-design-system/pull/259/commits/b78be73d8eb8f5536bf8b98c1f2d8be054897f7c this change work for you?

Reply:
yup